### PR TITLE
Add reflecting on the future post @randy

### PR DESCRIPTION
**Pull request by adamjberg**

### CODE REVIEW

File: _posts/randy/2022-06-24-reflecting-on-the-future.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+---
+layout: post
+title:  "Reflecting on the Future - A Mirror Sample of Thinks to Come"
+permalink: /@randy/reflecting-on-the-future
+ext_image: "assets/img/sample-in-progress.png"
+description: I am Old School human; equipped with analog gears but upgraded with digital belts. Having a fair amount of past, I spend a lot of time reflecting on the past. It seems like I always have a need to measure and gauge my present with the paths and events it took to get me here.
```
Comment (adamjberg) [inline _posts/randy/2022-06-24-reflecting-on-the-future.md:6]: image

This description shows up like this when sharing.  I think trimming down to a single sentence with a brief description would be good to do

File: _posts/randy/2022-06-24-reflecting-on-the-future.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+tags: tails analog-sampling
+---
+
+![](/assets/img/sample-in-progress.png)
+
+<!-- <video width="720" height="480" controls>
```
Comment (adamjberg) [inline _posts/randy/2022-06-24-reflecting-on-the-future.md:12]: As mentioned in message, I think MPEG-4/H.264 or webm should fix this issue

File: _posts/randy/2022-06-24-reflecting-on-the-future.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+   <source src="/assets/img/BlackHoleSon720.mp4" type="video/mp4">
+</video> -->
+
+I am Old School human; equipped with analog gears but upgraded with digital belts. Having a fair amount of past, I spend a lot of time reflecting on the past. It seems like I always have a need to measure and gauge my present with the paths and events it took to get me here. 
+
+I have a million memories of positive greatness, but the reflections that come back are often on the negative side. Perhaps that is just physics. A mirror, says the Blue Oyster Cult, is a negative space in a frame. We see everything backwards (and upside down, you know). The positive in life was expected; the negative was not.
```
Comment (adamjberg) [inline _posts/randy/2022-06-24-reflecting-on-the-future.md:18]: I'm guessing the Blue Oyster Cult reference is to https://www.youtube.com/watch?v=F1HrO2W6w-4 ?  If so, I think we should link.  Or honestly maybe just embed the youtube video.

File: _posts/randy/2022-06-24-reflecting-on-the-future.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+
+My present perspective is that I need to spend a lot more of my present tense time envisioning what my influence on the world's direction could be, so I can work more effectively now to radiate positive energy in the right direction. 
+
+Amid plenty of success, significant portions of my used-up lifetime and hard-earned money were wasted chasing down a few wrong avenues and dreams. Bleeding edge learning came with a costly curve - the urge has long hoped to bury those bodies and cover the trail leading to them. My family has instead encouraged me to go back inside my head and make sense of those experiences - good and bad, as they relate to our family business today. 
+
+Analog Sampling (or should it be Analog Dumpling: a bite sized pinch-wrapped morsel?) will thus forward be a digital (online) snippet of some analog (offline) part of the world as it happened to me. You probably don't need to hear the TLDR versions, so for the time sensitive audience we'll set the Frequency at weekly and the Amplitude at the 1,024-word stun level.
```
Comment (adamjberg) [inline _posts/randy/2022-06-24-reflecting-on-the-future.md:50]: Ah, I had read this paragraph I think before you made some edits.  This last bit makes a lot more sense now.

**Review by adamjberg — COMMENTED** [review]
Just a couple of requests here.  

For future articles, I think trying to break things up with sub headings will help the average distracted person keep their bearings through the article.

File: _posts/randy/2022-06-24-reflecting-on-the-future.md (hunk 3 above)
Comment (xyzrus) [reply _posts/randy/2022-06-24-reflecting-on-the-future.md:18]: good idea - option for lyrics as well https://www.azlyrics.com/lyrics/blueoystercult/mirrors.html

File: _posts/randy/2022-06-24-reflecting-on-the-future.md (hunk 2 above)
Comment (xyzrus) [reply _posts/randy/2022-06-24-reflecting-on-the-future.md:12]: https://user-images.githubusercontent.com/66967414/175633284-a2964a2f-9be7-42a2-af5d-a6ff1a6cd11d.mp4

Comment (xyzrus) [reply _posts/randy/2022-06-24-reflecting-on-the-future.md:12]: This is h.264 replacement file - will need renaming and relocation

File: _posts/randy/2022-06-24-reflecting-on-the-future.md (hunk 1 above)
Comment (xyzrus) [reply _posts/randy/2022-06-24-reflecting-on-the-future.md:6]: Card/desc looks good - does the latter appear in search?

**Pull request merged**